test/main.js

```diff
@@ -63,7 +63,66 @@ describe('Offline mode feature tests after a server restart', function() {
     this.timeout(20000);
     before(function () { return startRestbase(true); });
 
-    require('./features/pagecontent/idempotent')(config);
+    var assert = require('./utils/assert.js');
+    var preq = require('preq');
+
+    var offlineMessage = 'We are offline, but your request needs to be serviced online.';
+
+    describe('offline mode', function() {
+        it('should allow content revision retrieval from storage', function() {
+            this.timeout(20000);
+            return preq.get({
+                uri: config.bucketURL + '/Idempotent/html/76f22880-362c-11e4-9234-0123456789ab'
+            })
+            .then(function(res) {
+                assert.deepEqual(res.status, 200);
+            });
+        });
+        it('should not allow latest content retrieval from storage', function() {
+            this.timeout(20000);
+            return assert.fails(
```

Comment:
Because `.then(function(e)` would not run -- it would be shortcutted by the throwing of `e`, so we need to wrap the whole thing in a `try/catch`, which is what `assert.fails` helps us do.